Apps/frontend/src/OscilloscopeWebGl.ts

```diff
@@ -100,6 +105,31 @@ export class OscilloscopeWebGl {
       this.webgl.uniform1f(amplitudeUniform, channelAmplitude);
 
       this.webgl.drawArrays(this.webgl.LINE_STRIP, 0, samples.length);
+
+      // draw twice to make thicker lines
+      if (get(thicknessAdjustment)[i]) {
+        for (let j = -LINE_THICKNESS_DUPLICATES; j < 0; j++) {
+          offsetUniform = this.webgl.getUniformLocation(
+            this.channelProgram,
+            "u_offset"
+          );
+          channelOffset = get(offsetAdjustment)[i] + j * LINE_THICKNESS_DELTA;
+          this.webgl.uniform1f(offsetUniform, channelOffset);
+
+          this.webgl.drawArrays(this.webgl.LINE_STRIP, 0, samples.length);
+        }
+
+        for (let j = 1; j <= LINE_THICKNESS_DUPLICATES; j++) {
+          offsetUniform = this.webgl.getUniformLocation(
+            this.channelProgram,
+            "u_offset"
```

Comment:
As far as I can tell we only offset in y-direction, IMO we should also think about adding an offset to the x-direction (for more on this, refer to my review main comment).